src/validations/rules/ssp.sch

```diff
@@ -523,7 +523,15 @@
                 test="true()">A FedRAMP SSP must implement a statement for each of the following lettered response points for required controls: <sch:value-of
                     select="$implemented/@statement-id" />.</sch:report>
         </sch:rule>
-        
+        <sch:rule 
```

Comment:
You encountered a bothersome condition which I repeatedly had last year when multiple PRs were in flight.

This commit is the best way to fix such (when such a commit is sufficient).